### PR TITLE
Feature/adf-pipeline-resources

### PR DESCRIPTION
**Pull request by mehdi-kimakhe-amido**
📲 What
Adding ADF resources creating the example pipeline.

🤔 Why
So we can have an example data pipeline for users and also to be able to work on deployment pipelines

🛠 How
Using ADF interface and the link with Github.

👀 Evidence
N/A

🕵️ How to test
N/A

✅ Acceptance criteria Checklist
- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: data_factory/adf_managed/linkedService/ls_AzureSql_ExampleSource.json
[the remark this answers is not in the document]
Comment (mehdi-kimakhe-amido) [reply data_factory/adf_managed/linkedService/ls_AzureSql_ExampleSource.json:1]: Good spot Rishi, I think we'll need to create a SQL Server part of this repo and point the LS to that, so everything is in the same place.

Then we can delete the other SQL Server instance.

[the remark this answers is not in the document]
Comment (Trishisingh) [reply data_factory/adf_managed/linkedService/ls_AzureSql_ExampleSource.json:1]: Speaking with satendar, we will use this as is i.e in key vault, we might take this in phase 2 to have sql provisioned with TF

**Review by Trishisingh — APPROVED** [review]
Looks okay to me

**Pull request merged**